apisix/admin/resource.lua

```diff
@@ -57,11 +57,21 @@ function _M:check_conf(id, conf, need_id)
     core.log.info("conf  : ", core.json.delay_encode(conf))
 
     -- check the resource own rules
-    return self.checker(id, conf, need_id, self.schema)
+    local ok, err = self.checker(id, conf, need_id, self.schema)
+
+    if not ok then
+        return ok, err
+    else
+        return need_id and id or true
+    end
 end
 
 
 function _M:get(id)
+    if self.unsupported_methods and core.table.array_find(self.unsupported_methods, "get") then
```

Comment:
Is it enough to use `if core.table.array_find(self.unsupported_methods, "get") then` ?

Reply:
@soulbird Done.